src/gateway/configure/dynamic-plugin-ordering/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Dynamic Plugin Ordering
+badge: enterprise
+content_type: explanation
+---
+
+The order in which plugins are executed in Kong is determined by their
+`static priority`. As the name suggests, this value is _static_ and can't be
+ easily changed by the user.
+
+You can override the priority for any Kong plugin using each plugin's
+`ordering` field. This determines plugin ordering during the `access` phase,
+and lets you create _dynamic_ dependencies between plugins.
+
+## Concepts
+
+### Dependency tokens
+
+Use one of the following tokens to describe a dependency to a plugin:
+
+* `before`
+	 The plugin will be executed _before_ a specified plugin or list of plugins.
+
+* `after`
+	 The plugin will be executed _after_ a specified plugin or list of plugins.
+
```

Comment:
Not sure if my suggestion is entirely correct - can you list out multiple plugins? eg:

```
PluginA:
  ordering:
    before:
      access:
        - PluginB
        - PluginC
        - PluginD
```
Or can you only set one plugin?

Reply:
multiple plugins can be set